starboard/shared/starboard/player/filter/testing/BUILD.gn

```diff
@@ -17,12 +17,6 @@ if (current_toolchain == starboard_toolchain) {
     # build_loader = false
     testonly = true
 
-    # TODO(b/384819454): Make these test cases work and re-enable associated
-    # tests.
-    if (target_platform == "linux-x64x11") {
-      defines = [ "SKIP_PROPRIETARY_CODEC_TESTS" ]
-    }
-
     sources = [
       "//starboard/testing/fake_graphics_context_provider.cc",
       "//starboard/testing/fake_graphics_context_provider.h",
```

Comment:
I think we should still uncomment `player_components_test.cc` and
`video_decoder_test.cc` below in l.43-44?

```
      # TODO(b/384819454): Audit tests are re-enabled as unit tests.
      #"player_components_test.cc",
      #"video_decoder_test.cc",
```

Unless they're broken or you're planning to do this later.

Reply:
Ah I see it's #5677 . Never mind this.